### PR TITLE
Write pod logs as string to the file

### PR DESCRIPTION
**Pull request by vanajamukkara**
Currently YAML representation of pod log is written, fixing that to write the string without getting the YAML representation.

Now, the pod logs looks like - 
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests/1159/artifact/RESULT_ROOT/diagnosticlogs/ItMiiDomain/testCreateMiiDomain/ns-efgl.pod.domain1-admin-server.container.weblogic-server.log

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests/1159/artifact/RESULT_ROOT/diagnosticlogs/ItMiiDomain/testCreateMiiDomain/

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java [hunk 1]
```diff
@@ -268,11 +269,25 @@ public static void collectLogs(String namespace, String resultDir) {
...
+   * @throws IOException when write fails
+   */
+  private static void writeToFile(Object obj, String resultDir, String fileName, boolean asYaml)
+      throws IOException {
     LoggingFacade logger = getLogger();
     logger.info("Generating {0}", Paths.get(resultDir, fileName));
     if (obj != null) {
       Files.write(Paths.get(resultDir, fileName),
-          dump(obj).getBytes(StandardCharsets.UTF_8)
+          (asYaml == true ? dump(obj).getBytes(StandardCharsets.UTF_8) : ((String)obj).getBytes(StandardCharsets.UTF_8))
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java:290]: redundant check for boolean.

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java:290]: can u clarify where is the redundancy and what do u suggest

Comment (sankarpn) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java:290]:
```suggestion
          (asYaml ? dump(obj).getBytes(StandardCharsets.UTF_8) : ((String)obj).getBytes(StandardCharsets.UTF_8))
```

**Comment by tbarnes-us** [comment]
LGTM too. Cool.

**Comment by vanajamukkara** [comment]
> LGTM too. Cool.

Thanks @tbarnes-us . Can u please approve

**Pull request merged**